### PR TITLE
resources: show also audio titles (fixed #3548)

### LINKED ISSUE #3548: audiofile names dont have actual title
image

should present "beautiful random minor"

### PR DESCRIPTION
**Pull request by rlam20**
fixes #3548 
image
after fix it displays the name listed in library:
image

### CODE REVIEW

File: app/src/main/java/org/ole/planet/myplanet/base/BaseContainerFragment.kt [hunk 1]
```diff
@@ -103,10 +104,20 @@ abstract class BaseContainerFragment : BaseResourceFragment() {
     private fun openIntent(items: RealmMyLibrary, typeClass: Class<*>?) {
         val fileOpenIntent = Intent(activity, typeClass)
         if (items.resourceLocalAddress?.contains("ole/audio") == true || items.resourceLocalAddress?.contains("ole/video") == true) {
+            Log.d("openIntent", "Touched File Before Passing: ${items.resourceLocalAddress}")
```
Comment (ewhang5) [inline app/src/main/java/org/ole/planet/myplanet/base/BaseContainerFragment.kt:107]: I think log statements should be removed after finished debugging.

**Review by Okuro3499 — CHANGES_REQUESTED** [review]
myPlanet has a lite version and some files also need to be updated when you make changes to their full versions example the BaseContainerFragment has a lite so implement the same changes to it
https://github.com/open-learning-exchange/myplanet/tree/master/app/src/main/java/org/ole/planet/myplanet/base#:~:text=BaseActivity.kt-,BaseContainerFragment.kt,BaseContainerFragment.kt.lite,-BaseDialogFragment.kt

File: app/src/main/java/org/ole/planet/myplanet/base/BaseContainerFragment.kt (hunk 1 above)
Comment (Okuro3499) [reply app/src/main/java/org/ole/planet/myplanet/base/BaseContainerFragment.kt:107]: @rlam20 remove the logs

**Comment by Okuro3499** [comment]
there is a merge conflict

**Review by Okuro3499 — APPROVED** [review]

**Review by ewhang5 — APPROVED** [review]

**Review by dogi — COMMENTED** [review]
@rlam20 show both filename and resource title

**Review by dogi — APPROVED** [review]

**Pull request merged**